suite-native/module-home/src/components/EmptyAccountsState.tsx

```diff
@@ -0,0 +1,84 @@
+import { Dimensions } from 'react-native';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { Text, AlertBox, Card, Image, VStack, Button } from '@suite-native/atoms';
+import { useActiveColorScheme } from '@suite-native/theme';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Translation, useTranslate } from '@suite-native/intl';
+import {
+    AccountsImportStackRoutes,
+    HomeStackParamList,
+    HomeStackRoutes,
+    RootStackParamList,
+    RootStackRoutes,
+    StackToTabCompositeNavigationProp,
+} from '@suite-native/navigation';
+
+import { SettingsButtonLink } from './SettingsButtonLink';
+
+const SCREEN_HEIGHT = Dimensions.get('screen').height;
+
+const imageStyle = prepareNativeStyle(_ => ({
+    maxHeight: SCREEN_HEIGHT * 0.25,
+    width: '100%',
+    alignItems: 'center',
+}));
+
+const cardStyle = prepareNativeStyle(utils => ({
+    paddingTop: 40,
+    paddingBottom: utils.spacings.extraLarge,
+}));
+
+type NavigationProp = StackToTabCompositeNavigationProp<
+    HomeStackParamList,
+    HomeStackRoutes.Home,
+    RootStackParamList
+>;
+
+export const EmptyAccountsState = () => {
+    const { applyStyle } = useNativeStyles();
+    const navigation = useNavigation<NavigationProp>();
+
+    const { translate } = useTranslate();
+
+    const colorScheme = useActiveColorScheme();
+
+    const getImageSource = () => {
```

Comment:
I would prefer useMemo here than function, no need to create new function in every rerender and also kind of nicer than function since it's basically static value

Reply:
https://github.com/trezor/trezor-suite/pull/9694/commits/c9f9b0587b4ba3bea0645ae010e508e0e0911fc4